.travis.yml

```diff
@@ -49,7 +49,7 @@ branches:
 install:
  - true
 script:
-  - ./gradlew ciBuild -Dscan
+  - ./gradlew ciBuild -Dscan -s
```

Comment:
It's a separate commit which was added to get more information about Javadoc issues. Raised #786 for 2.x.